components/discover/ActionButton.tsx

```
@@ -0,0 +1,33 @@
+const ActionButton = ({ icon, label, onClick, activeColor }: ActionButtonProps) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleClick = () => {
+    setIsActive(!isActive);
+    if (onClick) {
+      onClick();
+    }
+  };
+
+  return (
+    <button onClick={handleClick} className="flex flex-col items-center gap-1 text-white">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)